README.md

```diff
@@ -1,44 +1,44 @@
-
-In order to start developing locally you need to clone your new repository to your local machine. Then open your favorite CLI and simply navigate to the folder containing your repository and start up gatsby or next.js in development mode with the following commands.
+To develop locally, clone your new repository to your local machine. Open the terminal, navigate to the folder containing your repository, and start up Gatsby or Next.js in development mode with the following commands.
 
 **Nextjs**
 
-```cli
+"`cli
 $ yarn install
 $ yarn dev
+
 ```
 
 **Gatsby**
 
-```cli
+"`cli
 $ npm install
 $ npm run develop
 ```
 
-Afther yarn is ready with setting up your dev environment you should be able to navigate to [http://localhost:3000/](http://localhost:3000/) on Next or to [http://localhost:8000/](http://localhost:8000/) for gatsby and see the same application as on
-[https://conductionnl.github.io/nl-design-skeleton-gatsby/](https://conductionnl.github.io/nl-design-skeleton-gatsby/).  Feel free to open up your project code and make some changes you should see the changes in your development environment isntantainusly.
+After Yarn has successfully set up your dev environment, navigate to [http://localhost:3000/](http://localhost:3000/) on Next.js or to [http://localhost:8000/](http://localhost:8000/) for Gatsby and see the same application as on
+[https://conductionnl.github.io/nl-design-skeleton-gatsby/](https://conductionnl.github.io/nl-design-skeleton-gatsby/). Feel free to open up your project code and make some changes. You should see the changes in your development environment instantaneously due to hot-reloading.
 
 ## Publishing your prototype to the internet (gatsby only)
 
-The gatsby version of the skeleton application has build in support for github pages, meaning that you can build your application into a static website and publish it as a github page. The skeleton repository comes with a build in github action for publishing itsself as a github page, meaning that you can have your prototype automatically published to the internet on a code push.
+The Gatsby version of the skeleton application has build-in support for GitHub pages, meaning you can build your application into a static website and publish it as a Github page. The skeleton repository comes with a build-in GitHub action for publishing itself as a GitHub page. You can have your prototype automatically published to the internet on a code push this way.
 
-For this to work you will need to do active github-pages on your repository, go to your repository settings and click on pages select gh-pages as a source and press on save (if you do not see a gh-pages branch yet you can create one by pushing to master).
+You need to activate GitHub pages on your repository for this to work. Go to your repository's settings, go to pages, select `gh-pages` as a source, and save (if you do not see a gh-pages branch, you can create one by pushing to main).
```

Comment:
"... repository for this to work." _this_ kan misschien wat duidelijker omschreven worden. E.g. "... on your repository for automatic deployment to GitHub pages to work."

De tweede gh-pages mag ook geschreven worden als `gh-pages`.

Ik begrijp uit deze tekst dat wanneer er nog geen `gh-pages` branch bestaat die aangemaakt wordt wanneer ik naar `main` push? Klopt dit? Lijkt mij namelijk logischer dat ik gewoon de branch `gh-pages` moet aanmaken, maar misschien zit ik er naast!

Reply:
gaat automatisch, heb jhet ook gecheckt. Dit stukje had iemand anderes eerst geschreven.

 Wel vaker GH pages gedaan, en nooit een specifieke branch moeten maken. Als je niet via Gatsby doet, moet je het volgens mij zelf ook updates specificeren.